oc-includes/osclass/classes/controller/admin/CAdminUsers.php

```diff
@@ -317,6 +320,27 @@ public function doModel()
                 osc_add_flash_ok_message($msg, 'admin');
                 $this->redirectTo(osc_admin_base_url(true) . '?page=users');
                 break;
+            case ('login'):
```

Comment:
login case is too generic make it more unique so it wouldn't confuse anyone.

Reply:
@navjottomer done